### ISSUE TITLE
Calibration upload a17fde

### ISSUE THREAD

**Issue by bronger** [lensfun/lensfun#1749]
Calibration images were uploaded to the directory that starts with “`a17fde_`”.

Please read the [workflow description](https://github.com/lensfun/lensfun/blob/master/tools/calibration_webserver/workflow.rst) for further instructions about the calibration.

Additional comments by the uploader:

> Sigma 24mm_1:2_DG_DN_f2.0

**Comment by tuxfanx** [comment]
the lens has just been added to the db by @uchrisu
@uchrisu: you generated the data with your awesome script lf_fitexif_se. The lens is also available for L-mount cameras. Would it make sense to add this mount to the db as well? I already know that a separate calibration would be better, but where do we end up then? I found some raw images made with an L-mount camera [here](https://www.dpreview.com/sample-galleries/8947078325/sigma-24mm-f2-dg-dn-contemporary-sample-gallery-dpreview-tv/3949872156). The distortion correction seems to work well. At tca I'm unsure seems to be small with the lens anyway.

**Comment by uchrisu** [comment]
I think we can add the L-Mount for now and cross-check it when there are additional samples available. I checked them with an image following your link and at least the distortion worked really good. The TCA is somehow difficult since it is very small. Personally I also suffer a little bit from color-blindness but I ask a friend to compare all three images (picture I took, upload a17fde and the one behind your link) and she said the corrected versions are all better than without correction, also for the L-mount version. So I think we can simply add the L-Mount for now to all mirrorless Sigma lenses.
